site/test/ColorCalculator.php

```diff
@@ -0,0 +1,79 @@
+<?php
+include('config/init.php');
+
+
+echo "
+	<form action='' method='get'>
+		<select name='Color1'>
+			<option value='Red'>
+			Red
+			</option>
+			<option value='Blue'>
+			Blue
+			</option>
+			<option value='Yellow'>
+			Yellow
+			</option>
+		</select>
+		+
+		<select name='Color2'>
+			<option value='Red'>
+			Red
+			</option>
+			<option value='Blue'>
+			Blue
+			</option>
+			<option value='Yellow'>
+			Yellow
+			</option>
+		</select>
+		<input type='submit' name='submit' />
+	</form>
+";
+
+// var_dump($_REQUEST);
+
+if(isset($_REQUEST['submit'])){
+	if($_REQUEST['Color1'] == 'Red'){
+		if($_REQUEST['Color2'] == 'Red'){
+			echo'<body style="background-color:red">
+				<h2>You mix red with red, you get red!</h2>';
+		}
+		elseif($_REQUEST['Color2'] == 'Blue'){
+			echo'<body style="background-color:purple">
```

Comment:
In order to reduce the amount of hmtl you're writing, it'd be better to save a `$BackgroundColor` variable (and I guess a message) and then insert them into one single `<body>` and `<h2>` at the bottom.

One of the big problems here is what happens if you need to make changes. Like let's say I asked you to make it so the background color is always white, but the text color is purple or whatever. You'd have to change every single one of these instead of just tweaking the single bit of html at the bottom.